megamek/src/megamek/server/Server.java

```diff
@@ -2268,7 +2268,7 @@ private void endCurrentTurn(Entity entityUsed) {
         boolean turnsChanged = false;
         boolean outOfOrder = false;
         GameTurn turn = game.getTurn();
-        if (game.isPhaseSimultaneous() && (entityUsed != null)
+        if (game.isPhaseSimultaneous()
```

Comment:
This might look like I removed a null check, but since it was being done twice in a row...